recipes/Orange3-Bioinformatics/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "Orange3-Bioinformatics" %}
+{% set version = "3.0.3" %}
+{% set sha256 = "5dd3f52cb8ad4f4edddabc25e754e65d6a1616958e440a87c647e02f3098d98c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests-cache
+  run:
+    - serverfiles
+    - requests
+    - requests-cache
+    - genesis-pyapi
```

Comment:
Are there any version requirements that upstream specifies for these?

Reply:
There are no versions specified in [requirements.](
https://github.com/biolab/orange3-bioinformatics/blob/master/requirements.txt)